poleno/utils/tests/test_templatetags.py

```diff
@@ -241,18 +254,33 @@ def test_change_lang_tag(self):
         Checking that it generates correct links to the same view in different languages.
         """
         lang = ((u'de', u'Deutsch'), (u'en', u'English'), (u'fr', u'Francais'))
-        with self.settings(LANGUAGES=lang): # Fix active languages
-            r1 = self.client.get(u'/en/language/')
-            r2 = self.client.get(u'/de/language/')
-            r3 = self.client.get(u'/fr/language/')
-
-            for r in [r1, r2, r3]:
-                self.assertIs(type(r), HttpResponse)
-                self.assertEqual(r.status_code, 200)
-
-            self.assertEqual(r1.content, u'(/en/language/)(/de/language/)(/fr/language/)')
-            self.assertEqual(r2.content, u'(/en/language/)(/de/language/)(/fr/language/)')
-            self.assertEqual(r3.content, u'(/en/language/)(/de/language/)(/fr/language/)')
+        with self.settings(
+                LANGUAGES=lang,
+                MIDDLEWARE_CLASSES=[mc for mc in settings.MIDDLEWARE_CLASSES
```

Comment:
@martinmacko47 Ak pribudne este jeden test, s rovnakymi settings, tak by som vytvoril novu spolocnu TestCase classu s `_pre_setup` a `_post_teardown`. Teraz by to este nemalo taky uzitok, kedze su tie testy iba dva.